### PR TITLE
Skylight

### PR DESCRIPTION
**Pull request by cdoyle-temple**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -38,15 +38,11 @@ gem "omniauth-rails_csrf_protection"
 gem "puma", "~> 6.4.0"
 gem "recaptcha", require: "recaptcha/rails"
 gem "simple_form"
+gem "skylight"
 gem "sprockets-rails"
 gem "stimulus-rails"
 gem "turbo-rails"
 
-group :production do
-  gem "dalli"
```
Comment (sensei100) [inline Gemfile:46]: @cdoyle-temple Are you sure this is not needed?  It is part of the memcached integration.

**Comment by sensei100** [comment]
@dkinzer You know more about the memcached integration than I do.  Can you confirm whether it is safe to remove dalli?

File: Gemfile (hunk 1 above)
Comment (cdoyle-temple) [reply Gemfile:46]: Not sure why my reply on another tab doesn't show up here, it's marked pending(?). Anyway
> @sensei100 David and I added them when trying to debug recaptcha. If memcached is still needed, and they are integral to it, then they should stay.

Also, we were not using it before the investigation so I don't think it's necessary.

**Comment by sensei100** [comment]
@cdoyle-temple If you are wanting to remove the whole memcached integration, then there is work to do in kubernetes to make that happen. I think it would be better if you left that in the Gemfile for now and created a new Jira ticket for the work of removing memcached.

**Comment by cdoyle-temple** [comment]
@sensei100 That makes sense. I was unaware of us adding anything to kubernetes in order to add those gems.

**Review by sensei100 — APPROVED** [review]

**Pull request merged**